go/pkg/client/client.go

```diff
@@ -78,7 +78,9 @@ type Client struct {
 	// ExecutableMode is mode used to create executable files.
 	ExecutableMode os.FileMode
 	// RegularMode is mode used to create non-executable files.
-	RegularMode     os.FileMode
+	RegularMode os.FileMode
+	// UtilizeLocality is to specify whether client downloads files utilizing disk access locality.
+	UtilizeLocality bool
```

Comment:
Please make it a client Opt, like ChunkMaxSize or StartupCapabilities.

Reply:
done